charts/podiumd/README.md

```diff
@@ -80,6 +80,22 @@
 | Open Notificaties | 1.7.1   |
 | Open Zaak         | 1.15.0  |
 
+### 2.0.2 (candidate)
+
+**PodiumD Helm chart version: 1.1.20**
+
+| Component         | Version |
+|-------------------|---------|
+| ClamAV            | 1.4.1   |
+| Keycloak          | 24.0.5  |
+| Objecten          | 2.4.4   |
+| Objecttypen       | 2.2.2   |
+| Open Formulieren  | 2.7.9   |
+| Open Inwoner      | 1.21.3  |
+| Open Klant        | 2.3.0   |
+| Open Notificaties | 1.7.1   |
+| Open Zaak         | 1.15.0  |
+
```

Comment:
Moet elastic hier ook aan toegevoegd worden? Ik zie ook dat die kiss-elastic genoemd is op 87/103.
De naamgeving en inrichting van elastic lijkt nu wat vreemd, maar dat kunnen we misschienbeter in een losse PR rechttrekken

Reply:
Als het in de helm-chart staat, dan verwacht ik het ook in dit lijstje 👍

Reply:
Bedoel je:

```
  - name: kiss-elastic
    alias: kisselastic
    version: 1.0.0
    repository: "@kiss-elastic"
    condition: kisselastic.enabled
    tags:
      - contact
```
Die is dus NIET part of de release, dus waarom zou ik hem dan opnemen? Dan klopt het toch niet.



```
tags:
  portaal: false
  contact: false
  zaak: false
```

Reply:
Ik ben het eens met Sjoerd
Daarnaast is het lijstje wat genoemd staat het hoofdcomponenten lijstje wat Dimpact vaststelt over wat in de release zit
Aanvullende componenten, zoals redis/elastic/solr, zijn nodig om een applicatie te laten werken maar hoeft niet vermeld te worden mijns inzien